core/src/include/constants.hpp

```diff
@@ -31,6 +31,13 @@ const double Tt = 273.16;
 
 //! Ratio of circumference to radius
 const double tau = 6.28318530717958647652;
+
+// degrees to radians as a hex float
+const double deg2rad = 0x1.1df46a2529d39p-6;
+
+// radians to degrees
+// TODO: Rewrite as a hex float
+const double rad2deg = 1. / deg2rad;
```

Comment:
```suggestion
const double rad2deg = 0x1.ca5dc1a63c1f8p+5;
```

I looked it up on the internet (https://observablehq.com/@jrus/hexfloat)